firmware/config/engines/ford_aspire.cpp

```diff
@@ -19,24 +19,24 @@
-/* Load 1.840000 */{	-0.584000,	+7.658000,	+16.714001,	+23.254000,	+29.351999,	+30.978001,	+32.141998,	+30.874001,	+30.896000,	+26.507999,	+24.558001,	+24.389999,	+25.761999,	+35.492001,	+35.492001,	+35.492001},
-/* Load 2.053333 */{	-0.584000,	+7.862000,	+16.538000,	+23.254000,	+29.232000,	+31.296000,	+32.520000,	+30.142000,	+30.388000,	+25.903999,	+24.370001,	+24.082001,	+24.792000,	+24.351999,	+24.351999,	+24.351999},
-/* Load 2.266667 */{	+1.364000,	+7.726000,	+16.806000,	+23.254000,	+29.639999,	+31.006001,	+32.298000,	+30.912001,	+29.882000,	+26.392000,	+24.664000,	+27.233999,	+25.374001,	+25.417999,	+25.417999,	+25.417999},
-/* Load 2.480000 */{	-1.364000,	+10.490000,	+16.705999,	+22.441999,	+28.101999,	+30.238001,	+32.363998,	+30.719999,	+30.896000,	+26.608000,	+24.664000,	+24.431999,	+24.500000,	+25.510000,	+25.510000,	+25.510000},
-/* Load 2.693333 */{	-9.864000,	+10.416000,	+11.680000,	+19.150000,	+25.754000,	+27.936001,	+32.554001,	+30.656000,	+30.153999,	+27.184000,	+25.252001,	+22.812000,	+24.452000,	+25.219999,	+25.219999,	+25.219999},
-/* Load 2.906667 */{	-9.866000,	-5.452000,	-2.854000,	+17.212000,	+17.552000,	+20.688000,	+25.660000,	+27.809999,	+27.691999,	+27.224001,	+25.882000,	+25.360001,	+26.100000,	+27.992001,	+27.992001,	+27.992001},
-/* Load 3.120000 */{	-9.864000,	-5.452000,	-2.854000,	+0.342000,	+12.526000,	+16.218000,	+21.364000,	+27.590000,	+25.780001,	+24.170000,	+24.664000,	+25.584000,	+26.490000,	+31.968000,	+31.968000,	+31.968000},
-/* Load 3.333333 */{	-9.864000,	-5.516000,	-2.854000,	+0.226000,	+2.738000,	+3.816000,	+11.924000,	+18.808001,	+21.038000,	+21.538000,	+21.209999,	+22.228001,	+25.046000,	+25.156000,	+25.156000,	+25.156000},
-/* Load 3.546667 */{	-9.866000,	-5.518000,	-2.854000,	-0.000000,	+3.022000,	+3.816000,	+6.428000,	+7.788000,	+19.426001,	+20.860001,	+19.966000,	+21.030001,	+21.396000,	+21.570000,	+21.570000,	+21.570000},
-/* Load 3.760000 */{	-9.864000,	-5.516000,	-2.772000,	+0.226000,	+2.732000,	+3.500000,	+6.798000,	+8.102000,	+8.660000,	+9.500000,	+11.788000,	+20.132000,	+20.072001,	+20.510000,	+20.510000,	+20.510000},
-/* Load 3.973333 */{	-9.864000,	-5.518000,	-2.854000,	-0.000000,	+2.880000,	+3.816000,	+6.420000,	+8.320000,	+8.426000,	+8.532000,	+11.470000,	+11.442000,	+13.610000,	+12.022000,	+12.022000,	+12.022000},
-/* Load 4.186667 */{	-9.750000,	-5.518000,	-2.604000,	-0.000000,	+2.880000,	+3.654000,	+6.050000,	+6.888000,	+8.372000,	+9.364000,	+11.764000,	+11.732000,	+11.864000,	+12.376000,	+12.376000,	+12.376000},
-/* Load 4.400000 */{	-0.350000,	-5.590000,	-0.502000,	-0.910000,	-0.864000,	-0.954000,	-1.324000,	+7.436000,	-1.170000,	-1.054000,	-2.058000,	-2.098000,	-2.636000,	+12.352000,	+12.352000,	+12.352000}
+/* Load 1.200000 */{	-0,	+7,	+16,	+23,	+29,	+31,	+32,	+30,	+30,	+26,	+24,	+25,	+25,	+25,	+25,	+25},
```

Comment:
I do not get what's going on here. Are we simply loosing precision or are introducing some hard-coded packing/encoding?

Reply:
Simply throwing away the fractional part.  Nobody needs better than 1% VE and 1 degree timing accuracy, especially not in a base map.